src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -781,6 +854,16 @@ std::string postQueryContext
     }
   }
 
+  if ((ciP->apiVersion == V2) && (ciP->uriParamOptions["count"]))
```

Comment:
Based in our previous discussion, I'd suggest to add an explanatory comment here:

```suggestion
  // Before implementing pagination for CPrs, this block of code was part of step 02.
  // However, in that step we only have the count for CB entities. It is in the new location
  // (after queryForward() invocation) when we have the total count CB+CPr
  if ((ciP->apiVersion == V2) && (ciP->uriParamOptions["count"]))
```